ecommerce/frontend/src/components/Recommendations.js

```diff
@@ -0,0 +1,47 @@
+import './Recommendations.less'
+import { useEffect, useState } from 'react'
+import { formatProduct } from '../utils'
+import { notification, Spin } from 'antd'
+import { api } from '../api'
+import { ProductCard } from './product_card/ProductCard'
+import { useAppContext } from '../context/AppContext'
+
+export const Recommendations = () => {
+    const [isLoading, setIsLoading] = useState(true)
+    const [products, setProducts] = useState([])
+
+    useEffect(() => {
+        fetch()
+    }, [])
+
+    async function fetch() {
+        try {
+            setIsLoading(true)
+            const {
+                data: { status, products },
+            } = await api.get(`/recommendation`)
+            if (status.successful) {
+                setProducts(products.map(formatProduct))
+            } else {
+                notification.warning({ message: status.message })
+            }
+        } catch {
+            notification.error({ description: 'Failed to load recommendations' })
+        } finally {
+            setIsLoading(false)
+        }
+    }
+
+    return (
+        <Spin spinning={isLoading}>
+            <div className="recommendations-wrapper">
+                <h1 className="recommendations-header">Recommended Products For You</h1>
```

Comment:
Maybe you can write something more personalized like "We have selected these products for you Özdeniz!". I don't know the best way to write this but I think you might try to include the first name of the user. Feel free to not do it.

Reply:
I have added the name of the user to the header. 
![image](https://user-images.githubusercontent.com/44326869/105634540-86bf5900-5e6f-11eb-9dd6-107ea1c7d8ba.png)